src/main/java/org/benetech/servicenet/domain/view/ActivityInfo.java

```diff
@@ -67,4 +67,8 @@
     public Organization getOrganization() {
         return organizations.iterator().next();
     }
+
+    public Set<OrganizationMatch> getOrganizationMatches() {
```

Comment:
there is lombok getter and setter used in this class

Reply:
fixed